source/documentation/fuseki2/fuseki-configuration.md

```diff
@@ -56,7 +56,7 @@ Some useful prefix declarations:
     PREFIX fuseki:  <http://jena.apache.org/fuseki#>
     PREFIX rdf:     <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
     PREFIX rdfs:    <http://www.w3.org/2000/01/rdf-schema#>
-    PREFIX tdb1:    <http://jena.hpl.hp.com/2008/tdb#>
+    PREFIX tdb:     <http://jena.hpl.hp.com/2008/tdb#>
```

Comment:
I think it's better to keep `tdb1` and `tdb2` and use them consistently.

Reply:
The in-memory example needs updating as well.

Renaming section "TDB" as "TDB1" and swapping with "TDB2" would be good.
Using TDB2 is preferred. TDB1 is mostly legacy.

Reply:
That's a lot of changes I'm suggesting!

Here's every in one alternative PR:
https://github.com/apache/jena-site/pull/202

Reply:
OK, I'll close this one then.

Reply:
Thanks for the PR anyway. It needed to be fixed.